content/blog/2021/04/2021-04-07-contributhon-participants.adoc

```diff
@@ -0,0 +1,105 @@
+She loves to convert designs to exact replicas in the form of webpages and she likes to learn new things in her free time.
+
+You'll recognize her contributions as link:https://github.com/Esther101[`Esther101`] on GitHub and as link:https://issues.jenkins.io/issues/?jql=creator%20%3D%20%20esther101[`esther101`] on the link:https://issues.jenkins.io/issues/?jql=creator%20%3D%20%20esther101[Jenkins issue tracker].
+
+== Cynthia Iradukunda
+
+image:/images/post-images/2020-04-contributhon-participants/cynthia-iradukunda.png[Cynthia Iradukunda, width=250px]
+
+Cynthia Iradukunda is based in Kigali, Rwanda.
+She is currently a junior Computing student at the African Leadership University (ALU) in Mauritius.
+She wants to be a software engineer because it will help her solve real-world problems while also allowing her to use her coding skills.
+Her coding skills include but not limited to Java, JavaScript, and attention to detail.
+By contributing to the Git plugin, she hopes to help users have a smooth process using its documentation, get involved with the project, and connect with other community members.
+
+You'll recognize her contributions as link:https://github.com/ciradu2204[`ciradu2204`] on GitHub and as link:https://issues.jenkins.io/issues/?jql=creator%20%3D%20%20ciradu2204[`ciradu2204`] on the link:https://issues.jenkins.io/issues/?jql=creator%20%3D%20%20ciradu2204[Jenkins issue tracker].
+
+== Lucy Karimi
+
+// image:/images/post-images/2020-04-contributhon-participants/lucy-karimi.jpg[Lucy Karimi, width=250px]
+
+Lucy is based in Nairobi, Kenya.
+She is a passionate web and Android developer and willing to learn more.
+
+You'll recognize her contributions as link:https://github.com/luciahroyalty101[`luciahroyalty101`] on GitHub and as link:https://issues.jenkins.io/issues/?jql=creator%20%3D%20%20luciahroyalty[`luciahroyalty101`] on the link:https://issues.jenkins.io/issues/?jql=creator%20%3D%20%20luciahroyalty101[Jenkins issue tracker].
+
+== About the Contributhon projects
+
+See the link:/blog/2021/03/19/SheCodeAfrica-announcement/[previous blog post] for more information about SheCodeAfrica, the Contributhon, and the plans for Jenkins.
+
+* link:https://docs.google.com/document/d/1xhmEtwYIlGuuKtwn6Ek8DWyBEkPFA5q8CfWRRL7xZ9U/edit#[Jenkins development tasks]
+* link:https://sites.google.com/shecodeafrica.org/contributhon/mentor-orgs/accepted-organizations[SheCodeAfrica Contributhon mentoring organizations]
+* link:https://sites.google.com/shecodeafrica.org/contributhon[SheCodeAfrica Contributhon]
+* link:https://www.shecodeafrica.org/[SheCodeAfrica]
+
+Conversations related to the Contributhon are happening in a link:https://cdeliveryfdn.slack.com/archives/C01TDDNSC90[Continuous Delivery Foundation slack channel].
+
+== Mentors
+
+We're very grateful to the mentors from the Jenkins project that are hosting mentoring sessions, reviewing pull requests, and encouraging the mentees.
+Thanks to:
+
+* Meg McRoberts
```

Comment:
```suggestion
* link:https://github.com/StackScribe[Meg McRoberts]
```

Reply:
Good suggestion.  Will apply in the update that I submit with Lucy's picture and an improved bio.